distributed/protocol/tests/test_serialize.py

```diff
@@ -388,3 +389,27 @@ def _(x):
 
     header, frames = serialize([MyObj(), MyObj()])
     assert header["compression"] == [False, False]
+
+
+@pytest.mark.parametrize(
+    "test_pair",
+    [
+        ({i: i for i in range(10)}, False),
+        (set(range(10)), False),
+        (tuple(range(100)), False),
+        ({"x": MyObj(5)}, True),
+        ({"x": {"y": MyObj(5)}}, True),
+        pytest.param(
+            ([1, MyObj(5)], True),
+            marks=pytest.mark.xfail(reason="Only checks 0th element for now."),
+        ),
+        ([MyObj([0, 1, 2]), 1], True),
+        (tuple([MyObj(None)]), True),
+        ({("x", i): MyObj(5) for i in range(100)}, True),
+    ],
+)
+def test_check_dask_serializable(test_pair):
```

Comment:
And then use those terms below

```suggestion
def test_check_dask_serializable(data, is_serializable):
```